src/neo/SmartContract/Native/Tokens/Nep17Token.cs

```diff
@@ -67,10 +67,10 @@ internal protected virtual void Mint(ApplicationEngine engine, UInt160 account,
             state.Balance += amount;
             storage = engine.Snapshot.Storages.GetAndChange(CreateStorageKey(Prefix_TotalSupply), () => new StorageItem(BigInteger.Zero));
             storage.Add(amount);
-            PostTransfer(engine, null, account, amount);
+            PostTransfer(engine, null, account, amount, data);
```

Comment:
```suggestion
            PostTransfer(engine, null, account, amount, null);
```

We don't need data in Mint.

Reply:
For Nep17 not, maybe for NFT Mint yes.

Reply:
Done, also was removed from Burn

Reply:
Maybe it should had been kept for `Burn`, no?